roles/openshift_health_checker/library/aos_version.py

```diff
@@ -130,6 +130,14 @@ def _retrieve_available_packages(expected_pkgs):
 
         pkgs = list(aquery.filter(name=expected_pkgs))
 
+        if not pkgs:
+            # pkgs list is empty, raise because no expected packages found
+            raise AosVersionException('\n'.join([
+                'Unable to find any OpenShift packages.',
+                'Check your subscription and repo settings.',
+                str(excinfo),
```

Comment:
@ashcrow ha! sorry about that, totally lazy on my side. I just copied and didn't even notice that (interestingly, Syntastic didn't yell at me for it).